### PR TITLE
OPCT-28: Introduce mirror repository variable

### PR DESCRIPTION
**Pull request by rvanderp3**
Relates to https://github.com/redhat-openshift-ecosystem/provider-certification-tool/pull/45

### CODE REVIEW

**Review by mtulio — COMMENTED** [review]
Overall looks good. I will take a second look tomorrow with the cli's changes.

File: openshift-tests-provider-cert/plugin/executor.sh [hunk 1]
```diff
@@ -133,17 +142,19 @@ elif [[ -n "${CERT_TEST_FILE:-}" ]]; then
...
         | grep "${CUSTOM_TEST_FILTER_STR}" \
-        | ${UTIL_OTESTS_BIN} run -f - \
+        | ${UTIL_OTESTS_BIN} run  ${IMAGE_MIRROR} -f - \
         | tee -a "${RESULTS_PIPE}" || true
 
 # Default execution - running default suite.
 # Set E2E_SUITE on plugin manifest to change it (unset PLUGIN_ID).
 else
-    suite="${E2E_SUITE:-kubernetes/conformance}"
+    suite="${E2E_SUITE:-kubernetes/conformance}"
```
Comment (mtulio) [inline openshift-tests-provider-cert/plugin/executor.sh:154]:
```suggestion
    suite="${E2E_SUITE:-kubernetes/conformance}"
```

**Review by faermanj — APPROVED** [review]
lgtm :)

**Comment by mtulio** [comment]
@rvanderp3 on https://github.com/redhat-openshift-ecosystem/provider-certification-plugins/pull/36 we refacted the executor to remove some unused blocks and organize it to improve the maintenance. Would you mind rebasing and adapt to the new format?
Thanks

**Comment by rvanderp3** [comment]
/hold

rebased/refactored, placing a hold while i test

**Comment by mtulio** [comment]
I am ok to approve after your tests.
/lgtm

**Pull request merged**